### PR TITLE
Feature/icons

### LINKED ISSUE #70: Icons!!!
My thoughts are:

- Left arrow
- Right Arrow
- Journal Icon
- Update Icon
- Cancel Icon
- Save Icon

### PR DESCRIPTION
**Pull request by chrissycooper**
### Files Changed:
- most components had an icon added. 
- link to icon suite added to `index.html`

### What Has Changed: 
Icons added for journal button, back and forward arrows, etc

### Notes: (includes questions, ideas for the future, anything you want to call out)
in `journalEntry` there are variables that hold the icons for the buttons that were waiting for an incoming PR to be merged. 

### Closes: (any issues that it closes)
closes #70

### CODE REVIEW

File: client/public/index.html [hunk 1]
```diff
@@ -7,9 +7,9 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Playfair+Display+SC:ital,wght@1,700&family=Plus+Jakarta+Sans&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/83ada33ef5.js" crossorigin="anonymous"></script>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <link href="https://fonts.googleapis.com/css2?family=Caveat:wght@600&family=Noto+Sans&family=Raleway:wght@300;500&family=Roboto+Mono:wght@200;400&display=swap" rel="stylesheet">
```
Comment (jfogiato) [inline client/public/index.html:12]: YEET

**Review by jfogiato — APPROVED** [review]
Thanks for digging into this one, CRC! It may have been a bit tedious, but I think these tweaks will help pull the app together and make it feel way more cohesive!

File: client/src/Components/FeelingsLog/FeelingsLog.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ const FeelingsLog = ({logs}) => {
         <td>{log.date.toLocaleString(DateTime.DATE_MED)}</td>
         <td>{toTitleCase(log.feeling)}</td>
         <td>{log.activity}</td>
-        <td>{<Link to={`/how-you-felt/entry/${log.entry_id}`}>📓</Link>}</td>
+        <td>{<Link to={`/how-you-felt/entry/${log.entry_id}`}><i class="fa-solid fa-book"></i></Link>}</td>
```
Comment (espressoGoddess) [inline client/src/Components/FeelingsLog/FeelingsLog.js:24]: I'm curious why you used an i tag here! isn't this the arrow?

File: client/src/sampleData/activityData.js [hunk 3]
```diff
@@ -137,7 +137,7 @@ const activityData = [
   },
   {
     activityId: 28,
-    activity: 'write in your journal about what expectations you might have had and how your experience is differing from that projection.',
+    activity: 'Write in your journal about what expectations you might have had and how your experience is differing from that projection.',
```
Comment (espressoGoddess) [inline client/src/sampleData/activityData.js:140]: nice attention to detail

**Review by espressoGoddess — APPROVED** [review]
Thanks for making this happen! It looks so clean!

File: client/src/Components/FeelingsLog/FeelingsLog.js (hunk 2 above)
Comment (chrissycooper) [reply client/src/Components/FeelingsLog/FeelingsLog.js:24]: it is! good question. This is the html auto generated by Font Awesome, so I have no idea.

File: client/public/index.html (hunk 1 above)
Comment (KatherineBlaine) [reply client/public/index.html:12]: Beautiful thanks for cleaning that up!

File: client/src/Components/Activities/Activities.js [hunk 4]
```diff
@@ -28,9 +28,9 @@ const Activities = ({ activities, addEntry, setFeeling }) => {
   return (
     <section className="activity-page">
       <div className="activity-card">
-        {currentActivityIndex > 0 && <button className="activities-btns" onClick={prevActivity}>←</button>}
+        {currentActivityIndex > 0 && <button className="activities-btns" onClick={prevActivity}><i class="fa-solid fa-arrow-left-long"></i></button>}
```
Comment (KatherineBlaine) [inline client/src/Components/Activities/Activities.js:31]: LOVE

**Review by KatherineBlaine — APPROVED** [review]
Thanks for the great work! Excited to see how the icons look!

**Pull request merged**